ui/src/nav.tsx

```diff
@@ -32,6 +32,8 @@ export interface NavItem {
   icon?: S
   /** True if this item should be disabled. */
   disabled?: B
+  /** The item title, typically displayed as a tooltip. */
+  title?: S
```

Comment:
Should be called `tooltip` instead of `title` to be consistent with the rest of the components.